app/src/main/java/org/kinecosystem/kinit/network/TaskService.kt

```diff
@@ -64,6 +63,309 @@ class TaskService(context: Context, api: TasksApi,
         })
     }
 
+    val DummyTask = """
+{
+  "id": "0",
+  "start_date": 1532252893,
+  "provider": {
+    "name": "Kinit Team",
+    "image_url": "https://cdn.kinitapp.com/brand_img/poll_logo_kin.png"
+  },
+  "min_to_complete": 2,
+  "memo": "1-kit-sb0ea956eabef4705bd26b",
+  "tags": [
+    "Finance"
+  ],
+  "type": "quiz",
+  "title": "Kin Quiz",
+  "price": 50,
+  "desc": "Let's test your Kinowledge!",
```

Comment:
spelling

Reply:
probably intentional. kin-knowledge. get it?